mltsp/custom_feature_tools.py

```diff
@@ -445,10 +449,13 @@ def extract_feats_in_docker_container(container_name, path_to_tmp_dir):
     # Create container
     cont_id = client.create_container(
         "mltsp/extract_custom_feats",
-        volumes={"/home/mltsp": ""})["Id"]
+        volumes={"/home/mltsp": "",
+                 "/home/copied_data_files": ""})["Id"]
     # Start container
     client.start(cont_id,
-                 binds={cfg.PROJECT_PATH: {"bind": "/home/mltsp"}})
+                 binds={cfg.PROJECT_PATH: {"bind": "/home/mltsp", "ro": True},
+                        path_to_tmp_dir: {"bind": "/home/copied_data_files",
```

Comment:
Should this path be in `/home`, or would it be better in `/tmp/` or `/home/mltsp`?